### DISCUSSION TITLE
Change token_type and multi-tenancy issue

### DISCUSSION THREAD

**Discussion by huudufs** [ory/hydra#2795] [Q&A] [answered]
Hi Experts,

I have 2 issues need your support:
1. Can I change token_type from Bearer to Oauth (when I call /oauth2/introspect and it responsed Bearer) ?
2. Our project is using ory hydra, can you suggest ways to manage multi-tenant, can I use 1 ory hydra and multiple databases, each DB for 1 tenant, or use 1 DB and can distinguish the information of each tenancy stored in that DB.

Best Regards,

**Comment by vinckr** [comment, marked as answer]
Hello @huudufs 
Apologies for the late answer. 

1. Can you explain a little bit what you want to achieve? Are you looking for the authorization code grant type? This document might help: https://www.ory.sh/hydra/docs/advanced/
2. You can do both of these options. I would suggest this however:

> Write a “meta” consent app and redirect to the concrete consent app based on e.g. client id or some other discriminators

**Reply by vinckr** [reply]
Hey @huudufs ,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
